aroma/tests/test_features.py

```diff
@@ -2,49 +2,45 @@
 import os.path as op
 import pandas as pd
 
-import pytest
-
 from aroma import features
-from aroma.tests.utils import get_tests_resource_path
 
 
-def test_feature_time_series(nilearn_data):
+def test_feature_time_series(mel_mix, mc, max_correls):
 
-    # Get path to confounds file
-    test_path, _ = op.split(nilearn_data.func[0])
+    np.random.seed(1)
```

Comment:
Isn't there a way of making this fixed for the three functions, instead of repeating it in all of them?

Reply:
I don't know of any way to set a seed for all tests at once.